### PR TITLE
Add privacy policy, add footer to first page and link to github/privacy policy. Clean up FAQ and Tutorial.

### PR DESCRIPTION
**Pull request by pselvana**
Adding an initial privacy policy.  The policy is broader than the services we offer; it includes PII capture which we don't do.

Footer added:
image

Privacy Policy modal opened - click on link in Footer and close with "Close" button at bottom or `esc` key:
image
image

FAQ updated:
image

Updates to the top intro verbiage:
image

Update to Tutorial:
image

### CODE REVIEW

File: src/components/Intro.tsx [hunk 1]
```diff
@@ -57,8 +57,12 @@ export default function Intro() {
       <div className="container mx-auto flex items-center justify-between">
         <div className="mb-10">
           <p className="mt-4 text-lg">
-            Finding A Time is a simple tool that helps you schedule meetings
-            with your team, friends or clients.
+            We are the simplest, fastest and free way to schedule meetings with
+            your team, friends or clients.
+          </p>
+          <p className="mt-4 text-lg">
+            Their is no login required or account signup. Just create a new
```
Comment (ZhenyuYu1) [inline src/components/Intro.tsx]: "Their" should be "There".

File: src/components/FAQ.tsx [hunk 2]
```diff
@@ -12,6 +12,22 @@ export default function FAQ() {
       answer:
         'To invite participants, share the event link with them after creating a new event.',
     },
+    {
+      question:
+        'How do I find my created events or events I added my availability to?',
+      answer:
+        'Click on the "Dashboard" button in the top right corner of the screen to seee a list of your created events.',
```
Comment (ZhenyuYu1) [inline src/components/FAQ.tsx]: "Seee" should be "see".

Comment (pselvana) [reply src/components/FAQ.tsx]: Good catch. Updated.

File: src/components/Intro.tsx (hunk 1 above)
Comment (pselvana) [reply src/components/Intro.tsx]: Good catch. Updated.

**Pull request merged**